pkgs/os-specific/linux/kernel/linux-rpi.nix

```diff
@@ -2,8 +2,8 @@
 
 let
   # NOTE: raspberrypifw & raspberryPiWirelessFirmware should be updated with this
-  modDirVersion = "6.1.21";
-  tag = "1.20230405";
+  modDirVersion = "6.1.58";
+  tag = "stable_20231030";
```

Comment:
Unfortunately, `stable_20231123` is out now :/

Reply:
Updated to stable_20231123 for both firmwares and kernel.